src/components/DateTimePicker.js

```diff
@@ -156,7 +157,7 @@ class DatePicker extends React.Component {
     const spans = this.spans();
 
     return (
-      <Container fluid={true}>
+      <Container fluid={true} ref={this.props.dateTimePickerRef}>
         <Row>
           <Column span={spans.date}>
             <div
```

Comment:
This `div` already has a `ref` prop and it's the focusable element. Can you move the prop call there instead please.